frontend/src/app/business/business-settings/business-settings.component.html

```diff
@@ -23,7 +23,7 @@
             registerShopForm.get('shopDescription')?.touched
           ">{{ 'BUSINESS.REGISTRATION.VALIDATIONS.BUSINESS_DESCRIPTION' | translate }}</span>
     </div>
-    <div class="business-registration__field business-registration__field-middle">
+    <!-- <div class="business-registration__field business-registration__field-middle">
```

Comment:
if we don't need a code, let's delete it, we still have git to find it if we were in need